database/migrations/2021_03_06_183050_create_rates_table.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateRatesTable extends Migration
+{
+    public function up()
+    {
+        Schema::create('rates', function (Blueprint $table) {
+            $table->id();
+            $table->string('slug');
+            $table->unsignedInteger('amount');
```

Comment:
I miseed it on file - rate_category_id is missing.

Reply:
Done